src/views/Teams/Team.List.jsx

```diff
@@ -1,32 +1,61 @@
 import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
-import { getTeams } from '../../services/teams';
-
+import { deleteTeamById, getTeams } from '../../services/teams';
 
 function TeamList(){
   const [teams, setTeams] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
+  const loadTeams = async () => {
+    setIsLoading(true);
+    const teamsData = await getTeams();
+    setTeams(teamsData);
+    setIsLoading(false);
+  };
+
   useEffect(() => {
-    async function equipos(){
-      const teamsData = await getTeams();
-      setTeams(teamsData);
-      setIsLoading(false);
-    }
-    equipos();
+    loadTeams();
   }, []);
   
+  const handleDeleteTeam = async ({ id, name }) => {
```

Comment:
If there are any players on a team and I try to delete it, I'm not getting a user-facing error. You can handle this in a few ways: delete all players associated with a team when you delete the team, add an error for the user that they can't delete a team until the players are deleted, etc.